hdf/src/hcompri.c

```diff
@@ -108,7 +108,7 @@ HRPconvert(int32 fid, uint16 tag, uint16 ref, int32 xdim, int32 ydim, int16 sche
         HGOTO_ERROR(DFE_ARGS, FAIL);
 
     /* allocate special info struct for buffered element */
-    if ((info = HDmalloc((uint32)sizeof(crinfo_t))) == NULL)
+    if ((info = malloc((uint32)sizeof(crinfo_t))) == NULL)
```

Comment:
Possible LEAK if getting access_rec fails. info shoulld be freed under line 127 before ERROR macro is executed.